Makefile

```diff
@@ -35,16 +35,20 @@ VERSION_STRING=$(shell /usr/libexec/PlistBuddy -c "Print :CFBundleShortVersionSt
 
 all: build
 
-sourcery: Tests/LinuxMain.swift Source/SwiftLintFramework/Models/MasterRuleList.swift
+sourcery: Source/SwiftLintFramework/Models/MasterRuleList.swift Tests/SwiftLintFrameworkTests/AutomaticRuleTests.swift Tests/LinuxMain.swift
 
 Tests/LinuxMain.swift: Tests/*/*.swift .sourcery/LinuxMain.stencil
-	sourcery --sources Tests --templates .sourcery/LinuxMain.stencil --output .sourcery
+	sourcery --sources Tests --templates .sourcery/LinuxMain.stencil --output .sourcery --verbose --force-parse generated
```

Comment:
I had to add an extra extension to the generated file, otherwise there's no way to force Sourcery to parse it in order to generate LinuxMain.swift